pkg/server/options/options.go

```diff
@@ -171,6 +175,16 @@ func (o *Options) rawFlags() cliflag.NamedFlagSets {
 	fs.BoolVar(&o.Extra.ExperimentalBindFreePort, "experimental-bind-free-port", o.Extra.ExperimentalBindFreePort, "Bind to a free port. --secure-port must be 0. Use the admin.kubeconfig to extract the chosen port.")
 	fs.MarkHidden("experimental-bind-free-port") // nolint:errcheck
 
+	fs.StringSliceVar(&o.Extra.BatteriesIncluded, "batteries-included", o.Extra.BatteriesIncluded, fmt.Sprintf(
+		`A list of batteries included (= default objects that might be unwanted in production, but are very helpful in trying out kcp or for development). These are the possible values: %s.
```

Comment:
> These are the possible values: %s.

I find the message not very clear: the list just after contains the possible values that are not there by default.
Why don't we rather list the default values here, since the detailed list of all batteries (with description) is provided below ?

Reply:
no, the list is a complete list. The output of the default comes from pflag, not from us.